README.md

```diff
@@ -53,6 +53,7 @@ If you've come here to help contribute - Thanks! Take a look at the [contributin
   * [Function](#function)
     * [.toBeFunction()](#tobefunction)
     * [.toSatisfy(predicate)](#tosatisfypredicate)
+    * [.toSatisfyAll(predicate)](#tosatisfyallpredicate)
```

Comment:
Perhaps this should be under the `Array` section because it's a matcher on Array values.

I'm sure not sure if `toSatisfy(predicate)` should go under `Function` or just at the top under no category, do you have any ideas?

Reply:
Yup, thought so but then saw `.toSatisfy`, so decided to place it here. Moved now.